SparkPost/2web2ui
# SA-2253: Remove timezone validation

### What Changed

- Removed timezone validation

### How To Test

- Go to signals/analytics&timezone=Europe%2FLondon
- Also check signals/analytics&timezone=UTC
